src/javascript/static/pages/responsible_trading.js

```diff
@@ -0,0 +1,26 @@
+const State          = require('../../_common/storage').State;
+const Client         = require('../../app/base/client');
+const BinarySocket   = require('../../app/base/socket');
+const getElementById = require('../../../javascript/_common/common_functions').getElementById;
+
+const ResponsibleTrading = (() => {
+
+    const onLoad = () => new Promise(resolve => {
+        BinarySocket.wait('landing_company').then(() => {
```

Comment:
we should wait for authorize since that's what we're using, not landing company
```suggestion
        BinarySocket.wait('authorize').then(() => {
```

Reply:
fixed